app/growth_chart.py

```diff
@@ -56,24 +73,45 @@ def _plot_thread(self):
             except queue.Empty:
                 continue
             
-            fig = plt.figure(figsize=(20, 10))
+            fig = plt.figure(figsize=(20, 20))
+            fig.patch.set_facecolor('k')
             plt.plot(CHRONOLOGICAL_AGE[input.sex],
                      BONE_AGE[input.sex],
-                     color = 'g', marker='o', linestyle = '--',
+                     color = 'g', marker='o', markersize=14, linestyle = '--',
                      label = 'Brush Foundation Study')
-            f = scipy.interpolate.interp1d(BONE_AGE[input.sex], CHRONOLOGICAL_AGE[input.sex])
-            pred_int_x = f(input.bone_age)
-            plt.plot(pred_int_x, input.bone_age,
-                     color = 'r', marker = 'D', label = 'prediction')
-            # plt.plot(np.arange(0, max(chronological_age)+1), [pred_int] * (max(chronological_age)+1), color = 'b', linestyle = ':', label = 'prediction')
-            plt.xlabel('Chronological Age [years]')
-            plt.ylabel('Bone Age [months]')
-            plt.title('Bone Age Growth Chart')
+            plt.plot(CHRONOLOGICAL_AGE[input.sex],
+                     BOUND_LOW[input.sex],
+                     color = 'b', marker='o', markersize=14, linestyle = '--',
+                     label = 'Brush Foundation Study - lower bound')
+            plt.plot(CHRONOLOGICAL_AGE[input.sex],
+                     BOUND_HIGH[input.sex],
+                     color = 'b', marker='o', markersize=14, linestyle = '--',
+                     label = 'Brush Foundation Study- higher bound')
```

Comment:
@katarinachiam Should the label be 'Brush Foundation Study - upper bound' instead?

Reply:
ya i'll change! thanks for catching that